examples/medplum-demo-bots/src/epic/epic-query-patient.ts

```diff
@@ -0,0 +1,68 @@
+import { BotEvent, MedplumClient } from '@medplum/core';
+import { Patient } from '@medplum/fhirtypes';
+import { SignJWT } from 'jose';
+import { randomBytes, createPrivateKey } from 'crypto';
+import fetch from 'node-fetch';
+
+export async function handler(medplum: MedplumClient, event: BotEvent): Promise<Patient> {
+  const privateKeyString = event.secrets['EPIC_PRIVATE_KEY'] as string;
+  const clientId = event.secrets['EPIC_CLIENT_ID'] as string;
+  if (!privateKeyString || !clientId) {
+    return { resourceType: 'Patient' };
```

Comment:
Perhaps change the return type to `Promise<Patient | undefined>` and return `undefined` in this case?